dgraph/cmd/alpha/run.go

```diff
@@ -684,10 +684,13 @@ func setupServer(closer *z.Closer) {
 	baseMux.Handle("/probe/graphql", graphqlProbeHandler(gqlHealthStore, globalEpoch))
 
 	baseMux.HandleFunc("/admin", func(w http.ResponseWriter, r *http.Request) {
-		r.Header.Set("resolver", "0")
 		// We don't need to load the schema for all the admin operations.
```

Comment:
@aman-bansal  I am not very sure about this comment. I couldn't understand the purpose of lazy loading the schema here.

Reply:
Hmm.. possible. Could it be related to auth rules? Because I believe, to resolve getUser/getGroup/getRule, etc queries, GraphQL schema should not be needed.

Reply:
get group will need permissions. this permission set might be filtered using the current schema. i am not sure. just speculating

Reply:
there is /admin/schema api. do you think we should expose that api to override the schema if needed?

Reply:
Users can access `/admin/schema` even now. That can be used right now as well to fix the schema.

Reply:
Then this change is not needed.